webcomponents/bundle/src/main/bundle/sui-notifications.properties

```diff
@@ -8,3 +8,10 @@ no_notifications=No notifications
 connection_request_received=You received a connection request from {0}
 connection_request_accepted={0} accepted your connection request
 message_received={0} sent you a message
+assignmentCreated = Created/updated a new assignment "{0}" in "{1}"
```

Comment:
You need to use the same format for these key names as the rest of the file. So, you would use assignment_created here.

Reply:
Sure! Thanks for your review, Adrian. I was not aware of it, Lol.